### ISSUE TITLE
Post Content won't work

### ISSUE THREAD

**Issue by AndreeaCristinaRadacina** [Codeinwp/otter-blocks#1341]
### Description:
The Dynamic Value > Post Content inside the Custom Layouts won't display the content.

### How to reproduce:
1. Open the Custom Layouts > Template > Single Post
2. Introduce the Post Content
3. Check on the frontend

### Expected behaviour:
The content of the post is displayed.
### Current behaviour:
The content is not displayed.
### Reference:
https://www.loom.com/share/590351de83c147edb07df5e8652d3a71

### Technical info
* WordPress version: 6.1.1
* Plugin version: 2.1.3

**Comment by HardeepAsrani** [comment]
@AndreeaCristinaRadacina Have you tried it with something other than Post Content? Displaying Post Content inside Post Content isn't a practical usage as you will end up creating an infinite loop of Page Content rendering Page Content rendering Page Content... we all have seen Inception. :D

Let me know how it goes with something other than Post Content.

**Comment by AndreeaCristinaRadacina** [comment]
@HardeepAsrani I have tried to display Post Content inside the Single Post Template to build the desired appearance of the posts on the website. I have done this before with core block, and it looked like this:
![image](https://user-images.githubusercontent.com/86592992/205480741-68e4d739-56f3-47d9-a36b-fb6baefe987d.png)

Isn't this a use case of the Post Content?

**Comment by HardeepAsrani** [comment]
@AndreeaCristinaRadacina No, you're asking Post Content to Render itself. It's meant for Query or Single Post Templates, but not inside the Post Content itself. It has similar use cases but not to be used anywhere inside Post Content.
